optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/roster/Observable.java

```diff
@@ -0,0 +1,24 @@
+package org.optaplanner.openshift.employeerostering.gwtui.client.roster;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+class Observable {
+    Collection<Observer> observers;
```

Comment:
Why is the visibility (public, protected, private, friendly) of this field friendly and not private?

Reply:
I was gonna use java.util.Observable and java.util.Observer, but apparent gwt doesn't support it. Then I made Observable and Observer private inner classes of AbstractRosterView, but then AbstractRosterView couldn't implement Observer, so I extracted the code using Eclipse. I thought Eclipse would had made it public; will fix.